mne/utils/tests/test_check.py

```diff
@@ -102,3 +102,32 @@ def test_check_info_inv():
     picks = _check_info_inv(epochs.info, forward, noise_cov=noise_cov,
                             data_cov=data_cov)
     assert list(range(7, 10)) == picks
+
+
+def test_check_option():
```

Comment:
This is a good usecase to use pytest.mark.parametrize

Reply:
Thanks for pointing me to this function. However, using `pytest.mark.parametrize` ended up not being any shorter nor clearer than these four lines. A little too much changes across each line.

Reply:
...or I just don't understand the `parametrize` function too well. If you have some clever refactoring in mind, please share :)